config/postgresql/postgresql.go

```diff
@@ -0,0 +1,72 @@
+package postgresql
+
+import (
+	"database/sql"
+	"fmt"
+	"net/url"
+
+	"github.com/NYTimes/gizmo/config"
+)
+
+// Config holds everything yo need to
+// connect and interact with a Postgresql DB
+type Config struct {
+	User    string `envconfig:"POSTGRESQL_USER"`
+	Pw      string `envconfig:"POSTGRESQL_PW"`
+	Host    string `envconfig:"POSTGRESQL_HOST_NAME"`
+	Port    int    `envconfig:"POSTGRESQL_PORT"`
+	DBName  string `envconfig:"POSTGRESQL_DB_NAME"`
+	SSLMode string `envconfig:"POSTGRESQL_SSL_MODE"`
+}
+
+const (
+	// DefaultSSLMode is disabled
+	DefaultSSLMode = "disable"
```

Comment:
For security sake, I think we'd want SSL on by default and required users to change the config to make it insecure.